### PR TITLE
[MC 0.5][FEATURE] -  Wallet actions on Tab bar

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
This PR aims to implement a new tab bar option with the wallet actions

**Screenshots/Recordings**
* Tapping on the center option of the Tab bar should open the wallet actions sheet: https://recordit.co/efaM8kC6gu 

* Tapping on Buy from the wallet actions sheet should open the Buy flow: https://recordit.co/s05MkvgC0K

* Tapping on Swap from the wallet actions sheet should open the Swap flow: https://recordit.co/WoFElcZT0x

* Tapping on Receive from the wallet actions sheet should open the Receive flow: https://recordit.co/g824ov8Cm6

* Tapping on Send from the wallet actions sheet should open the Send flow: https://recordit.co/qhunztmLCs

* **Update small behaviour** -> closing action sheet after press any cta above: https://recordit.co/lBEkB0ZuBD

* Analytics working -> https://recordit.co/57rkZF2pTi

**Issue**

Progresses #

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by tommasini** [comment]
Onboarding wizard, it's okay: https://recordit.co/IPlRPzzVJL
Legacy and new toasts: https://recordit.co/13HP2FMsoQ

There is a little bit more of margin-bottom on the new toasts, do you know if this was intended @Cal-L ?

**Comment by Cal-L** [comment]
> Onboarding wizard, it's okay: https://recordit.co/IPlRPzzVJL
> 
> Legacy and new toasts: https://recordit.co/13HP2FMsoQ
> 
> 
> 
> There is a little bit more of margin-bottom on the new toasts, do you know if this was intended @Cal-L ?
> 
> 

For the new toast, it's normal since it respects both the bottom notch and the curvature of the bottom edges. We should probably push the legacy toast up a bit. If it's too complicated, I'd leave it alone since we'll refactor that anyways.

As for the onboarding wizard, can we push down the step that points at the browser?

For the wallet actions sheet, the padding on the top between the notch and the first option looks like it could use some padding.

**Comment by tommasini** [comment]
Thanks @Cal-L for the review and eagle eyes!

Onboarding wizard browser step fixed:


Onboarding wizard edit account name step fixed:


Padding on send while it just has two options:



The legacy toast padding seems to not be that easy, let's save it for future refactoring to the new Toast 🙏

File: app/components/Views/WalletActions/WalletActions.tsx [hunk 1]
```diff
@@ -51,6 +51,18 @@ const WalletActions = () => {
   const swapsIsLive = useSelector(swapsLivenessSelector);
   const dispatch = useDispatch();
 
+  // Dismiss the sheet when Android back button is pressed.
+  useEffect(() => {
```
Comment (tommasini) [inline app/components/Views/WalletActions/WalletActions.tsx:55]: Amazing catch, I fixed that on ToU and do not remembered this possibility here

**Review by Cal-L — COMMENTED** [review]
Left some comments

File: app/component-library/components/Navigation/TabBar/TabBar.tsx [hunk 2]
```diff
@@ -42,33 +50,65 @@ const TabBar = ({ state, descriptors, navigation }: TabBarProps) => {
...
         }
       };
 
+      const isWalletAction = rootScreenName === Routes.MODAL.WALLET_ACTIONS;
+      const iconProps = {
+        size: isWalletAction ? AvatarSize.Md : AvatarSize.Lg,
+        backgroundColor: isWalletAction
+          ? colors.primary.default
+          : importedColors.transparent,
+        color: isWalletAction
```
Comment (sethkfman) [inline app/component-library/components/Navigation/TabBar/TabBar.tsx:78]: Can this be broken into a function? The nested ternaries make things a bit hard to follow

**Review by Cal-L — APPROVED** [review]
LGTM

File: app/component-library/components/Navigation/TabBar/TabBar.tsx (hunk 2 above)
Comment (tommasini) [reply app/component-library/components/Navigation/TabBar/TabBar.tsx]: Done <3

**Review by SamuelSalas — APPROVED** [review]
